src/commands/api-mesh/set-log-forwarding.js

```diff
@@ -0,0 +1,151 @@
+/*
+Copyright 2021 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+const { Command } = require('@oclif/core');
+const {
+	initSdk,
+	initRequestId,
+	promptConfirm,
+	promptSelect,
+	promptInput,
+	promptInputSecret,
+} = require('../../helpers');
+const logger = require('../../classes/logger');
+const { ignoreCacheFlag, autoConfirmActionFlag, jsonFlag, destinations } = require('../../utils');
+const { setLogForwarding, getMeshId } = require('../../lib/devConsole');
+
+class SetLogForwardingCommand extends Command {
+	static flags = {
+		ignoreCache: ignoreCacheFlag,
+		autoConfirmAction: autoConfirmActionFlag,
+		json: jsonFlag,
+	};
+
+	static enableJsonFlag = true;
+
+	async run() {
+		await initRequestId();
+
+		logger.info(`RequestId: ${global.requestId}`);
+
+		const { flags } = await this.parse(SetLogForwardingCommand);
+
+		const ignoreCache = await flags.ignoreCache;
+		const autoConfirmAction = await flags.autoConfirmAction;
+
+		let destinationConfig;
+		try {
+			destinationConfig = await this.inputAndValidateConfigs(destinations);
+		} catch (error) {
+			this.error(error.message);
+			return;
+		}
+		const { imsOrgCode, projectId, workspaceId, workspaceName } = await initSdk({
+			ignoreCache,
+		});
+
+		let meshId = null;
+
+		try {
+			meshId = await getMeshId(imsOrgCode, projectId, workspaceId, workspaceName);
+		} catch (err) {
+			this.log(err.message);
+			this.error(
+				`Unable to get mesh ID. Check the details and try again. RequestId: ${global.requestId}`,
+			);
+		}
+
+		// mesh could not be found
+		if (!meshId) {
+			this.error(
+				`Unable to get meshId. No mesh found for Org(${imsOrgCode}) -> Project(${projectId}) -> Workspace(${workspaceId}). Check the details and try again.`,
+			);
+		}
```

Comment:
The code from line 67-71 seems redundant to me because if getMeshId() fails you have a catch block which handles the error. Also the error messaging between the two is different. Can we remove the bottom logic